test/DynamoCoreWpfTests/WorkspaceSaving.cs

```diff
@@ -553,6 +563,85 @@ public void CanSaveAndReadWorkspaceName()
             Assert.AreEqual(fileName, ViewModel.Model.CurrentWorkspace.Name);
         }
 
+        [Test]
+        [Category("UnitTests")]
+        public void BackUpSaveDoesNotChangeName()
```

Comment:
I am revisiting the idea of disabling changing name for backup, anything going to hurt there? Because from the result, backup file will not be exactly same with regular saved which is probably fine but I want to confirm the `WHY` part.

Reply:
because backup files are saved to a different path than regular saves - thus the backup workspace name being changed based on the backup path is useless.

Reply:
disabling changing this will actually make the backup the same as the regular save - not the other way around.

Reply:
Thanks LGtM